src/overlay/StellarXDR.h

```diff
@@ -8,6 +8,4 @@
 namespace stellar
```

Comment:
This namespace is empty now, so it can be removed.

Reply:
removed